src/Components/Myprofile.js

```diff
@@ -0,0 +1,13 @@
+import './Myprofile.css';
+
+const Myprofile = () => (
+  <div className="queryresult">
+    {/* Selected Missions */}
+    under construction Rockets
```

Comment:
Just a suggestion, I would remove line 5, and make line 6 an `<h1> or <h2>` tag;
Following the page layout the 'My missons' and 'My rockets' title are not center but that one thing that can be changed later;
For now look good, you can merge :white_check_mark: